source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRRenderer.mm

```diff
@@ -198,16 +198,21 @@ + (UIView *)render:(UIView<ACRIContentHoldingView> *)view
     UIView *prevStretchableElem = nil, *curStretchableElem = nil;
 
     auto firstelem = elems.begin();
-    for(const auto &elem:elems)
+    auto prevElem = *firstelem;
+    
+    for (const auto &elem:elems)
```

Comment:
>: [](start = 25, length = 1)

nit: put spaces around the `:` here

Reply:
thanks

---
In reply to: [280538099](https://github.com/Microsoft/AdaptiveCards/pull/2829#discussion_r280538099) [](ancestors = 280538099)